### DISCUSSION TITLE
createArrowFiles run time

### DISCUSSION THREAD

**Discussion by janekim319** [GreenleafLab/ArchR#937] [Questions / Documentation]
I'm trying to run createArrowFiles() on 10x genomics fragment file of 11,000 cells. It was running, but it was still running after 1 hour. When I was following the tutorial, the three fragment files that added up to 10,000 cells total only took about 10 minutes and I'm unsure if my runtime of 1 hr + for one file of 11,00 cells is normal. I'm using addArchRThreads(threads = 4).

**Comment by rcorces** [comment]
no that does not sound normal. However, you can check if the tmp files are being written to which would tell you if any computation is still happening.

**Reply by janekim319** [reply]
I think it may have had to do with memory. When I shut down all the background apps open and only let R run, it ran in 25 minutes! Thank you!
